core/admin/EE_Admin_List_Table.core.php

```diff
@@ -735,7 +739,7 @@ public function get_hidden_columns()
      */
     public function single_row_columns($item)
     {
-        [$columns, $hidden, $sortable, $primary] = $this->get_column_info();
+        list($columns, $hidden, $sortable, $primary) = $this->get_column_info();
```

Comment:
Is something wrong with the [] syntax? Curios to know why do you reverted to the classic style? 🤔

Reply:
my IDE converts some of these things automatically, but this branch will be getting merged into master which still allows PHP 5.6, but PHP 5.6 does not allow array destructuring nor null coalescing operators